### PR TITLE
Adding SSp_Detector recipe

### PR DESCRIPTION
**Pull request by sjg2203**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from an official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sjg2203** [comment]
@conda-forge/help-python Ready for review!

**Comment by sjg2203** [comment]
@conda-forge/help-python Ready for review!

**Comment by sjg2203** [comment]
@conda-forge-admin (@ocefpaf) & @conda-forge/help-python Ready for review!

File: recipes/SSp_Detector/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "SSp_Detector" %}
+{% set version = "2023.10.19a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/SSp_Detector/meta.yaml:9]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/SSp_Detector/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  host:
+    - pip
+    - python >=3.10
+
+  run:
+    - python >=3.10
+    - mne
+    - numpy
+    - outdated
+    - scipy
```
Comment (ocefpaf) [inline recipes/SSp_Detector/meta.yaml:27]:
```suggestion
    - scipy >=1.11
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]
Please don't remove recipes. You should rebase or start from a fresh clone to avoid that. Also, when possible please use `grayskull pypi --strict-conda-forge SSp_Detector` to create the recipe. It produces a recipe that is almost merge-ready and much easier to review.

**Comment by sjg2203** [comment]
Hi @ocefpaf, should be ready for review now!
Thanks for the help

**Comment by sjg2203** [comment]
@conda-forge-admin (@ocefpaf) & @conda-forge/help-python @conda-forge/staged-recipes Ready for review!

File: recipes/SSp_Detector/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - tk
+
+test:
+  imports:
+    - SSp_Detector
```
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/SSp_Detector/meta.yaml:31]: Why not the pip check?

**Comment by ocefpaf** [comment]
Please do note remove an existing recipe. Rebase to fix that or send a fresh PR if that is easier for you.

**Pull request closed without merging**